### DISCUSSION TITLE
DirectorySeparator Windows vs Mac issue

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#908] [Ideas]
I have been exploring using the Mac and VS to develop on Oqtane.

The first issue I came in to was the file path separator in the ModuleCreator. Using the constant string value of "\\\\" will break the code as the file path separator for the Mac is "/" when the module is created.

Solution would be to use the .net constant Path.DirectorySeparatorChar using System.IO

Scanning the Solution for this constant "\\\\" resulted 7 counts in the Clients and Server projects.

**Comment by jimspillane** [comment]
We may want to consider using Path.AltDirectorySeparatorChar.  In the [Remarks for DirectorySeparatorChar](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.directoryseparatorchar?view=net-5.0#remarks), they suggest using the AltDirectorySeparatorChar for multiple platform development.
